File: ClientApp/src/app/models/respondent.ts
import { Feedback } from "./feedback";
import { Guid } from "guid-typescript";

export class Respondent {
  id?: string;
  mos: string;
  firstName: string;
  lastName: string;
  email: string;
  roleId: number;
  feedbacks: Feedback[];

  constructor() {
    this.id = Guid.create().toString();
    this.mos = "";
    this.firstName = "";
    this.lastName = "";
    this.roleId = 0;
    this.email = "";
  }
}

export interface RespondentField {
  key: string;
  type: string;
}

File: Models/Respondent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SurveyTool.Models
{
    public partial class Respondent
    {
        public Respondent()
        {
            Feedback = new HashSet<Feedback>();
        }

        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }
        public string Mos { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int RoleId { get; set; }

        public virtual Role Role { get; set; }
        public virtual ICollection<Feedback> Feedback { get; set; }
    }
}
